content/03.categorize.md

```diff
@@ -50,7 +50,7 @@ focus on barriers to achieving these goals. We also highlight approaches that
 researchers are taking to address challenges within the field, particularly with
 regards to data availability and labeling.
 
-### Imaging applications in healthcare
+### Imaging based patient categorization
```

Comment:
I prefer leaving this sub-heading as `Imaging applications in healthcare`.  Some of the applications, like anatomical localization, are not strictly about patient categorization.

Reply:
Agree!